### DISCUSSION TITLE
What are deep link specifications for desktop applications?

### DISCUSSION THREAD

**Discussion by stefandunca** [WalletConnect/walletconnect-monorepo#3978] [Developer Support] [answered]
Should I expect the pairing/authentication URI after the prefix (`myApp` example)?
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/47554641/d32397e7-f2be-489b-a4f8-c10a5f8b466c)

Can we test the deep linking in development? Is there any testing environment to provide custom deep link generation?

**Comment by glitch-txs** [comment, marked as answer]
these are for web app wallets. You can add a custom wallet to Web3Modal for testing

**Comment by stefandunca** [comment]
Some details on how to setup a debugging custom wallet: https://docs.walletconnect.com/web3modal/javascript/options#customwallets
